src/python/pants/backend/jvm/tasks/jvm_compile/class_not_found_error_patterns.py

```diff
@@ -8,31 +8,31 @@
 CLASS_NOT_FOUND_ERROR_PATTERNS = [
   # javac errors.
   (r'\s*\[error\] (?P<filename>\S+):(?P<lineno>\d+):(\d+): cannot find symbol\n'
-   '\s*\[error\]   symbol:   class (\S+)\n'
-   '\s*\[error\]   location: package (\S+)\n'
-   '\s*\[error\] import (?P<classname>\S+);'),
+   r'\s*\[error\]   symbol:   class (\S+)\n'
+   r'\s*\[error\]   location: package (\S+)\n'
```

Comment:
These two changed lines are not equivalent to the original! Remove the `\n` from both, though, and they are the same. 

I think the author likely intended for the final string to end in `\\n`, given they did so in line 10?